protocol/avdl/keybase1/common.avdl

```diff
@@ -179,6 +182,11 @@ protocol Common {
       array<PerUserKey> perUserKeys;
   }
 
+  record UserOrTeamLite {
+      UserOrTeamID id;
+      string username;
```

Comment:
No, it should be the team/subteam name.  Want me to rename it "name"?  I wasn't sure what to do, since it's part of the regular Keybase username namespace (similarly with things like `libkb.NormalizedUsername`).